### PR TITLE
Adds bottle tests to github actions

### PR DESCRIPTION
**Pull request by Genegrady**
## Description
Added a bottle.yml file to workflows in order to test the tracer against the library's ootb test suite

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Comment by brettlangdon** [comment]
Looks like a handful of failing bottle tests (22), but it doesn't seem like they are (all) related to the tracer being added?

File: .github/workflows/bottle.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      #Taken from install script inside of .github/workflows of test suite
+        run: |
+          for name in waitress "cherrypy<9" cheroot paste tornado twisted diesel meinheld\
+            gunicorn eventlet flup bjoern gevent aiohttp-wsgi uvloop; do
+            pip install $name || echo "Failed to install $name with $(python -V 2>&1)" 1>&2
+          done
+      - name: Inject ddtrace
+        run: pip install ../ddtrace
+      - name: Run tests
+        run: ddtrace-run pytest test
```
Comment (P403n1x87) [inline .github/workflows/bottle.yml:45]: Missing newline at EOF :)

File: .github/workflows/bottle.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          ref: master
+          path: bottle
+      - name: Install pytest
+        run: pip install -U pip pytest coverage
+      - name: Install Dependencies
+        run: sudo apt-get install -y libev-dev
+      - name: Install further dependencies
+        run: pip install mako jinja2
+      - name: Test dependencies (Server back-ends and template engines)
+      #Taken from install script inside of .github/workflows of test suite
```
Comment (brettlangdon) [inline .github/workflows/bottle.yml:33]: can you provide a comment with a link to where you grabbed this from.

File: .github/workflows/bottle.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        run: |
+          for name in waitress "cherrypy<9" cheroot paste tornado twisted diesel meinheld\
+            gunicorn eventlet flup bjoern gevent aiohttp-wsgi uvloop; do
+            pip install $name || echo "Failed to install $name with $(python -V 2>&1)" 1>&2
+          done
+      - name: Inject ddtrace
+        run: pip install ../ddtrace
+      - name: Run tests
+        run: ddtrace-run pytest test --continue-on-collection-errors -v
```
Comment (brettlangdon) [inline .github/workflows/bottle.yml:42]: can you add a comment on why we need `--continue-on-collection-errors` here?

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**